yt_dlp/downloader/hls.py

```diff
@@ -185,15 +90,15 @@ def is_ad_fragment_end(s):
                     if format_index and discontinuity_count != format_index:
                         continue
                     if frag_index > 0:
-                        self.report_error(
-                            'Initialization fragment found after media fragments, unable to download')
+                        # self.report_error(
+                        #    'Initialization fragment found after media fragments, unable to download')
```

Comment:
Is it okay to comment this warning? As far as I know it's non fatal

Reply:
Are you sure this doesn't break any streams? I'd rather have yt-dlp error out that download an unusable stream

Reply:
cc @shirt-dev since u added this

Reply:
It returns False in the following line, signaling an error. That's why I think it's safe, but I could raise an exception

Reply:
This corrupts the entire stream last time I checked. I think it is safe to completely ignore the media initialization and not download the initialization if it's already been downloaded before the media fragments.